cpp/src/structure/graph.cu

```diff
@@ -32,7 +34,9 @@ void degree_from_offsets(vertex_t number_of_vertices,
 }
 
 template <typename vertex_t, typename edge_t>
-void degree_from_vertex_ids(edge_t number_of_edges,
+void degree_from_vertex_ids(const cugraph::experimental::Comm& comm,
```

Comment:
If we are passing comm here, why do we have https://github.com/rapidsai/cugraph/pull/840/files#diff-ba59ce02957abb53c35f1a1929fc0374R64

Reply:
`degree_from_vertex_ids` is not a member of `GraphBase` while `comm` is set in `GraphBase`.  Hence we need to pass it somehow from the Graph to this stand-alone function. Does this answer your question?

Reply:
I see, I thought degree_from_vertex_ids is a member function, and thanks for the explanation!